src/index.html

```diff
@@ -35,6 +35,13 @@
     </div>
 
     <img class="made-by" src="assets/dash/2x/made by google@2x.png"/>
+    <div class="callout-oss">
+        <p>Dart is free, and open source
```

Comment:
```suggestion
        <p>Dart is free and open source
```

I was unsure about the comma, so I checked the Flutter page. It doesn't have one ("It’s free and open source."), and we should probably follow its lead.

Reply:
Sounds good.

Should there be a hyphen in 'open source', like: https://en.wikipedia.org/wiki/Free_and_open-source_software ?

Reply:
Based on my experiences, people will likely be more familiar with "open source" without a dash.

Reply:
No hyphen, but it's a good question. Google has a policy of not using a hyphen for "open source", even when  it's a verb or it precedes a noun (which this one doesn't): https://developers.google.com/style/word-list#letter-o.